src/CardList/CardList.js

```diff
@@ -50,8 +56,9 @@ function CardList({ cards, onListEdit }) {
     >
       Delete selected
     </button>
+    <button onClick={addCardButtonHandler}>Add a new card</button>
```

Comment:
The idea is you need to show modal after clicking this button and then to add it after clicking OK button on modal. Or handle cancel of adding.